core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -467,6 +473,24 @@ public <T> ServerBuilder childChannelOption(ChannelOption<T> option, T value) {
         return this;
     }
 
+    /**
+     * Sets the {@link Consumer} that customizes the Netty {@link ChannelPipeline}.
+     * This customizer is run right after the initial set of {@link ChannelHandler}s are configured.
+     * This customizer is no-op by default.
+     *
+     * <p>Note that usage of this customizer is an advanced
+     * feature and may produce unintended side effects, including complete
+     * breakdown. It is not recommended if you are not familiar with
+     * Armeria and Netty internals.
+     */
+    @UnstableApi
+    public ServerBuilder childChannelPipelineCustomizer(
+            Consumer<? super ChannelPipeline> childChannelPipelineCustomizer) {
+        requireNonNull(childChannelPipelineCustomizer, "childChannelPipelineCustomizer");
+        this.childChannelPipelineCustomizer = childChannelPipelineCustomizer;
```

Comment:
Should we make this additive? e.g.
```suggestion
        this.childChannelPipelineCustomizer =
             
   this.childChannelPipelineCustomizer.andThen(childChannelPipelineCustomizer);
```

Reply:
It is probably better than current code.
I'll fix it! Thank you.

Reply:
I've revised like this.